tests/std/tests/P0024R2_parallel_algorithms_transform_reduce/test.cpp

```diff
@@ -144,6 +146,25 @@ void test_case_incorrect_special_case_reasoning() {
     assert(transform_reduce(begin(c), end(c), d, 0, multiplies<>{}, plus<unsigned char>{}) == 0);
 }
 
+void test_case_narrowing_conversion() {
+    size_t a[]         = {1, 2, 3};
+    auto return_itself = [](size_t a) { return a; };

```

Comment:
Style nitpick: It's somewhat confusing for `return_itself`'s parameter to have the same name as the `a` array immediately above. Any other name like `x` would be clearer.